src/chapter5/challenges.md

```diff
@@ -1 +1,50 @@
 # Challenges
+
```

Comment:
Try and come up with new challenges. The old repo is not going to be used anymore so they're are not relevant.